src/main/kotlin/org/wfanet/measurement/integration/common/InProcessMeasurementReachAccuracyTest.kt

```diff
@@ -0,0 +1,283 @@
+
+  private val publicMeasurementsClient by lazy {
+    MeasurementsGrpcKt.MeasurementsCoroutineStub(inProcessCmmsComponents.kingdom.publicApiChannel)
+  }
+  private val publicMeasurementConsumersClient by lazy {
+    MeasurementConsumersGrpcKt.MeasurementConsumersCoroutineStub(
+      inProcessCmmsComponents.kingdom.publicApiChannel
+    )
+  }
+  private val publicCertificatesClient by lazy {
+    CertificatesGrpcKt.CertificatesCoroutineStub(inProcessCmmsComponents.kingdom.publicApiChannel)
+  }
+  private val publicEventGroupsClient by lazy {
+    EventGroupsGrpcKt.EventGroupsCoroutineStub(inProcessCmmsComponents.kingdom.publicApiChannel)
+  }
+  private val publicDataProvidersClient by lazy {
+    DataProvidersGrpcKt.DataProvidersCoroutineStub(inProcessCmmsComponents.kingdom.publicApiChannel)
+  }
+  private val publicRequisitionsClient by lazy {
+    RequisitionsGrpcKt.RequisitionsCoroutineStub(inProcessCmmsComponents.kingdom.publicApiChannel)
+  }
+
+  @Before
+  fun startDaemons() {
+    inProcessCmmsComponents.startDaemons()
+    initMcSimulator()
+  }
+
+  private fun initMcSimulator() {
+    val measurementConsumerData = inProcessCmmsComponents.getMeasurementConsumerData()
+    val eventQuery =
+      MetadataSyntheticGeneratorEventQuery(
+        SyntheticGenerationSpecs.POPULATION_SPEC,
+        InProcessCmmsComponents.MC_ENCRYPTION_PRIVATE_KEY
+      )
+    mcSimulator =
+      MeasurementConsumerSimulator(
+        MeasurementConsumerData(
+          measurementConsumerData.name,
+          InProcessCmmsComponents.MC_ENTITY_CONTENT.signingKey,
+          InProcessCmmsComponents.MC_ENCRYPTION_PRIVATE_KEY,
+          measurementConsumerData.apiAuthenticationKey
+        ),
+        OUTPUT_DP_PARAMS,
+        publicDataProvidersClient,
+        publicEventGroupsClient,
+        publicMeasurementsClient,
+        publicMeasurementConsumersClient,
+        publicCertificatesClient,
+        RESULT_POLLING_DELAY,
+        InProcessCmmsComponents.TRUSTED_CERTIFICATES,
+        eventQuery,
+        NoiseMechanism.CONTINUOUS_GAUSSIAN
+      )
+  }
+
+  @After
+  fun stopEdpSimulators() {
+    inProcessCmmsComponents.stopEdpSimulators()
+  }
+
+  @After
+  fun stopDuchyDaemons() {
+    inProcessCmmsComponents.stopDuchyDaemons()
+  }
+
+  private fun getReachVariance(measurementInfo: MeasurementInfo, reach: Long): Double {
+    val liquidLegionsSketchMethodology =
+      LiquidLegionsSketchMethodology(
+        RoLlv2ProtocolConfig.protocolConfig.sketchParams.decayRate,
+        RoLlv2ProtocolConfig.protocolConfig.sketchParams.maxSize,
+      )
+    val reachMeasurementParams =
+      ReachMeasurementParams(
+        StatsVidSamplingInterval(
+          measurementInfo.measurementSpec.vidSamplingInterval.start.toDouble(),
+          measurementInfo.measurementSpec.vidSamplingInterval.width.toDouble()
+        ),
+        DpParams(OUTPUT_DP_PARAMS.epsilon, OUTPUT_DP_PARAMS.delta),
+        StatsNoiseMechanism.GAUSSIAN
```

Comment:
Comparing to line 131, I am guessing StatsNoiseMechanism.Gaussian is for discrete Gaussian. Is it?